css/question.less

```diff
@@ -18,6 +18,15 @@
     }
   }
 
+  .sticky-question-header {
+    top: 10px !important;
```

Comment:
I guess you've tried to workaround the fact that react-sticky [doesn't support margins](https://github.com/captivationsoftware/react-sticky#overview--basic-example). However, I'd say it doesn't work too nicely for me in practice. The header is jumping and feels a bit weird. If I remove that style and top margin from `.question-header`, then it's smooth and nice.
So, I guess it would be better to change layout a bit, so the header doesn't require any margin styles (probably remove padding from `.question` and add some question-content container with padding instead).